_inc/client/plans/index.jsx

```diff
@@ -3,29 +3,81 @@
  */
 import React from 'react';
 import { connect } from 'react-redux';
+import { get } from 'lodash';
 
 /**
  * Internal dependencies
  */
-import { getSiteConnectionStatus } from 'state/connection';
 import PlanGrid from './plan-grid';
+import { SingleProductBackup } from './single-product-backup';
 import QuerySite from 'components/data/query-site';
+import { getSiteRawUrl } from 'state/initial-state';
+import { getAvailablePlans, getSitePlan } from 'state/site';
 
 export class Plans extends React.Component {
 	render() {
+		const { sitePlans, siteRawUrl } = this.props;
+
 		return (
-			<div>
+			<>
```

Comment:
What's the deal with these empty elements? Can't we just use React.Fragment?

Reply:
Or is this shorthand for that?

Reply:
`<>` is the new syntax as of React 16.2, but enough Jetpack devs have been disturbed by the `<>` that I'm changing it back 😆

Reply:
This is a 2-year old syntax FWIW 😉 I'm fine with using it, as long as our current build process supports it.

Reply:
I don't mind as long as it's the standard. I just hadn't seen it before.